### ISSUE TITLE
Trusty Tahr (Ubuntu 14.04) doesn't support ia32-libs

### ISSUE THREAD

**Issue by bsudy** [maestrodev/puppet-android#21]
The module doesn't work on Trusty Tahr because of the lack of ia32-libs package.

http://askubuntu.com/questions/107230/what-happened-to-the-ia32-libs-package

**Comment by apeeters** [comment]
A naive solution:
```Puppet
exec { '/usr/bin/dpkg --add-architecture i386 && apt-get update':
	unless => '/bin/grep -q i386 /var/lib/dpkg/arch',
	before => Class['android'],
}
```

**Comment by carlossg** [comment]
Fixed in #35
